nixos/modules/system/boot/systemd.nix

```diff
@@ -725,6 +725,8 @@ in
 
       "systemd/system.conf".text = ''
         [Manager]
+        DefaultCPUAccounting=yes
+        DefaultBlockIOAccounting=yes
```

Comment:
`BlockIOAccounting` refers to the legacy cgroup hierarchy (while `IOWeight` used above belongs to the unified hierarchy).

The way I read the manual, this is strictly redundant: enabling a type of resource control for one unit is said to imply enabling it for the slice it runs in and all parent slices.